src/psyclone/core/access_type.py

```diff
@@ -75,6 +76,10 @@ class AccessType(Enum):
     #: is available at compile-time and can be used for type properties such
     #: as kinds or dimensions.
     CONSTANT = 10
+    #: Is the output of a MIN reduction (i.e. global minimum value).
+    MIN = 11
+    #: Is the output of a MAX reduction (i.e. global maximum value).
+    MAX = 12
```

Comment:
Again MIN/MAX is not an AccessType (same with the pre-existing SUM). It is even more confusing that this refer to the output scalar, which is just written to. So why do we differentiate them?